### ISSUE TITLE
Disable direct commits - require a PR

### ISSUE THREAD

**Issue by rosskevin** [apollographql/react-apollo#1463]
@james I don't have admin privileges on `react-apollo` (which is ok), but I want to disable direct commit without a PR.  I see an issue that popped up about lodash https://github.com/apollographql/react-apollo/issues/1445 and related https://github.com/apollographql/react-apollo/issues/1286 where a fix was reverted in a direct commit without explanation https://github.com/apollographql/react-apollo/commit/557e4104704a8032e3c9b68cb00b0e32f4d2b7de

TL;DR - lets force PRs for commits so context is added (or can be added later if missing), tests are run etc.

**Comment by rosskevin** [comment]
Looks like this is already done and I missed it.  My mistake.
